arch/arm/soc/st_stm32/stm32l0/soc_gpio.c

```diff
@@ -135,6 +135,11 @@ int stm32_gpio_enable_int(int port, int pin)
 		return -EINVAL;
 	}
 
+	/* Port H EXTI should be enabled by value 0x5 instead of 0x7 */
+	if (port == STM32_PORTH) {
+		port = STM32_PORTF;
```

Comment:
Please use LL_SYSCFG_EXTI_PORTH (requires #include stm32l0xx_ll_system.h) or SYSCFG_EXTICR1_EXTI0_PH (no additional include requested).
Some future L0 chip could own PORT F.

Reply:
That is reasonable, thanks. Will do.

Reply:
Done.